client/apps/edit/components/admin/icons.jade

```diff
@@ -0,0 +1,2 @@
+.caret
+  include ../../public/icons/edit_caret_down.svg
```

Comment:
Yes have been thinking about this too -- would be great to find a way to load these files directly to the component, rather than using jquery to parse the jade file.  Craig mentioned a library for this, will look into it in another PR